Scripts/script-MD2HTML.yml

```diff
@@ -0,0 +1,29 @@
+commonfields:
+  id: MD2HTML
+  version: -1
+name: MD2HTML
+script: |-
+  from markdown2 import Markdown
+
+  markdowner = Markdown(extras=["tables"])
+  file_entry = demisto.args().get('file_id', '')
```

Comment:
changing the default to '' is redundant in that case since the argument is required.